docs/source/markdown/podman-volume-create.1.md

```diff
@@ -129,6 +129,77 @@ All containers are assigned larger project IDs (e.g. >= 100000).
 All volume assigned project IDs larger project IDs starting with 200000.
 This prevents xfs_quota management conflicts with containers/storage.
 
+## MOUNT EXAMPLES
+
+podman volume create allows the `type`, `device`, and `o` options to be passed to `mount(8)` when using the `local` driver.
```

Comment:
```suggestion
The `podman volume create` command allows the `type`, `device`, and `o` options to be passed to `mount(8)` when using the `local` driver.
```

Reply:
I updated the way it was written under QUOTAS , and under my changes.

Reply:
@WesselAtWork, good point about "The ... command".  It is a bit redundant, and when I took a second look at the whole page, I realized we'd either need to drop it here or add it in a number of other places to be consistent.  Once added, it would be TERRIBLY redundant.  Great pushback; please do drop my "The ... command" and it looks like you have. :+1: